plugins/omar-oms-plugin/grails-app/services/omar/oms/ChipperService.groovy

```diff
@@ -6,96 +6,114 @@ import javax.imageio.ImageIO
 
 import org.springframework.util.FastByteArrayOutputStream
 
-@Transactional(readOnly=true)
-class ChipperService {
+@Transactional(readOnly = true)
+class ChipperService
+{
 
-    def getTile(ChipperCommand cmd) {
-      
-log.info '*' * 50
-log.info cmd as String
-log.info'*' * 50
+    def getTile(ChipperCommand cmd)
+    {
 
-      HashMap chipperOptions = cmd?.toChipperOptions()
-      HashMap result = [:]
-      HashMap chipperResult = [:]
-      String outputFormat = cmd.outputFormat?:"image/png"
-      def hints = [type:outputFormat]
+        log.info '*' * 50
```

Comment:
Is "info" the appropriate level here? I don't think we need the stars. Instead, I suggest we prepend the 'cmd' with some unique message that we can grep/search. I.E. "Running getTile with ChipperCommand: <cmd>"